### ISSUE TITLE
Error importing tenacity on python 3.7.3

### ISSUE THREAD

**Issue by anudeepsamaiya** [jd/tenacity#166]
```
>>> import tenacity
        from tenacity.async import AsyncRetrying
                      ^
SyntaxError: invalid syntax
```

**Comment by jd** [comment]
Which version of tenacity are you using?

**Comment by anudeepsamaiya** [comment]
I was using tenacity-4.9.0. It got fixed in 5.0.4. Thanks.

**Comment by Priyanshu1381** [comment]
> I was using tenacity-4.9.0. It got fixed in 5.0.4. Thanks.

How did you upgrade it?
